### DISCUSSION TITLE
v6.0.0-alpha

### DISCUSSION THREAD

**Discussion by mzur** [mzur/gnome-shell-wsmatrix#171] [General]
This release is compatible with GNOME 40. It includes a huge overhaul of the code.

**Please report any bugs or discuss features in the discussion of this release.**

Alpha version known issues:

- The workspace grid in the overview can have the wrong ordering.
- The styling/spacing in the switcher popup could be better.
- When the switcher is opened with <kbd>Super</kbd>+<kbd>W</kbd>, keyboard shortcuts are not working (only the mouse).
- The mouse sometimes overrides the keyboard navigation if it happens to hover over the switcher.
- Moving a window between workspaces sometimes plays an animation and sometimes not.
- The "Next/Previous Bordered" mode does not work as expected.

<hr /><em>This discussion was created from the release <a href='https://github.com/mzur/gnome-shell-wsmatrix/releases/tag/v6.0.0-alpha'>v6.0.0-alpha</a>.</em>

**Comment by deekej** [comment]
Hello @mzur, to answer your question from my latest comment about the issue... Yes, I was using the latest version at the time. Now I couldn't replicate it at home, where I'm just using my laptop screen. However, I recall that this was happening where I was in the office, where I use multiple external monitors, and I often plug/unplug my laptop via Thunderbolt 3 Docking Station. I guess this re-plugging might be started of the blank desktop issue I've seen before... :)

EDIT: Double checking the version, it's possible I was really behind the PR's code. Now I'm using Github CLI to checkout the latest code instead of git aliases, so take my comment with a grain of salt, please. :)

**Reply by mzur** [reply]
Maybe you can try it in the office with the latest code sometime.
